frontend/integration-tests-cypress/cypress/tests/project_crud.spec.ts

```diff
@@ -0,0 +1,91 @@
+import { 
+  appHost, 
+  timeouts 
+} from '../support/utils';
+
+const getElemByDataId = dataId => cy.get(`[data-test-id="${dataId}"]`);
```

Comment:
This is a dup of https://github.com/openshift/console/pull/4831/files#diff-9a0a3b4e2af0d952c4d1f2a9b3d71072R3

Reply:
Yup, exactly.
I've been thinking about this.  Do we want to make selectors:
- more clear `getElemByDataId`?
- more easy to write, as perhaps just `getElem`?
- write several initially:  `elem.byData()`, and `elem.byCSS()`?